made authenticated user check case insensitive

jlaacke opened the pull request:
LDAP login is not case sensitive, so a user may successfully login but cannot be identified when user id in LDAP differs. As a result the user is logged in, but has no profile information (like mail address) and is not assigned to any LDAP groups. Therefore the check has to be case insentive too.

koevskinikola commented:
Hi @jlaacke ,

Thank you for the contribution! I started looking into it. 

We are doing a minor release this month and we have more or less defined what it will deliver. So it's possible that your contribution might be skipped for this version and be merged with the next one. I'll keep you updated on our decisions.

However, I noticed that you don't provide a test case to reproduce the problem that you're fixing. Would you be willing to create one?

Best,
Nikola

jlaacke commented:
Hi Nikola,

thanks for looking at my pull request. ;)

You are absolutely right, I added two test cases. One to proof, that login itself is case insensitive (which is also successful without my fix) and the other to ensure that filtering for logged in user also is case insensitive (which is what I fixed).

Cheers,
Jonas

koevskinikola commented:
Hey Jonas,

That's great! Thank you.

At the moment we can't merge it with our codebase since it has already been decided what will be included/fixed by the minor release at the end of this month.

However, I'll be extremely happy if your contribution is still available for a merge after the release. Would it be a problem for you to wait that long?

Best,
Nikola

jlaacke commented:
That's absolutely fine, I will keep my custom code just until it's merged. ;)

Cheers,
Jonas

koevskinikola commented:
Hi @jlaacke,

I finally merged your pull request with the master. :)

Thank you for the contribution!

Best,
Nikola
